SU2_CFD/src/transfer_physics.cpp

```diff
@@ -130,17 +138,25 @@ void CTransfer_FlowTraction::GetDonor_Variable(CSolver *flow_solution, CGeometry
   // Pinf: Pressure_infinite
   // div_vel: Velocity divergence
   // Dij: Dirac delta
+  // area: area of the face, needed if we transfer stress instead of force
   su2double Pn = 0.0, div_vel = 0.0;
   su2double Viscosity = 0.0;
   su2double Tau[3][3] = {{0.0, 0.0, 0.0},{0.0, 0.0, 0.0},{0.0, 0.0, 0.0}};
   su2double Grad_Vel[3][3] = {{0.0, 0.0, 0.0},{0.0, 0.0, 0.0},{0.0, 0.0, 0.0}};
   su2double delta[3][3] = {{1.0, 0.0, 0.0},{0.0,1.0,0.0},{0.0,0.0,1.0}};
+  su2double area = 0.0;
 
   su2double Pinf = flow_solution->GetPressure_Inf();
 
   Point_Flow = flow_geometry->vertex[Marker_Flow][Vertex_Flow]->GetNode();
   // Get the normal at the vertex: this normal goes inside the fluid domain.
   Normal_Flow = flow_geometry->vertex[Marker_Flow][Vertex_Flow]->GetNormal();
+  
+  if (consistent_interpolation)
+    for (iVar = 0; iVar < nVar; ++iVar) area += Normal_Flow[iVar]*Normal_Flow[iVar];
```

Comment:
Should be nDim in place of nVar, should't it?

Reply:
The whole CTransfer family works like that I think, nVar gets the value of nDim in the constructor I believe. I kept the convention.